jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/cloudsql/DisableOrEnableDBFlagsRule.java

```diff
@@ -49,12 +48,12 @@ public RuleResult execute(Map<String, String> ruleParam, Map<String, String> res
         }
 
         if (!StringUtils.isNullOrEmpty(vmEsURL)) {
-            vmEsURL = vmEsURL + "/gcp_cloudsql_sqlserver/_search";
+            vmEsURL = vmEsURL + "/"+dataBaseType+"/_search";
```

Comment:
The value of ES index should be read from ruleParam, it should be ruleParam.get(databaseType)

Reply:
String dataBaseType=ruleParam.get(PacmanRuleConstants.DB_TYPE); is the same